### ISSUE TITLE
Games that won't reach "Network Initialized" in-game

### ISSUE THREAD

**Issue by MojoJojoDojo** [hrydgard/ppsspp#14624]
### What happens?
Found 2 games that won't reach the "Network Initialized" part and either throw you back into a previous menu or stay frozen.
For EA Replay it gets stuck on the menu.
For Samurai Warriors - State Of War it will return to the previous menu.
Possibly other games got that issue but I dont remember which now.
It won't matter if you got WLAN off or on the result will be the same for all games.
Screenshot for EA Replay:
![image](https://user-images.githubusercontent.com/33392696/126069275-64d7c260-bdc0-4351-885f-7f91ab233141.png)
### What *should* happen?
You should proceed to the next menu and be able to play.
#### What hardware / device and operating system?
OS: Windows 10
CPU:Intel 8550U
#### What graphics card (GPU) or mobile phone model?
GPU:Radeon 530
#### What PPSSPP version (standalone/official), and did it work before?
Even before the latest 1.11.3 stable release they didn't work.
#### Which game or games?

ULUS10140
ULUS10089
#### Checklist

 - [ X] Test in the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already fixed.
 - [ X] [Search for other reports](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same issue.
 - [ X] Try resetting settings or older versions and include if the issue is related.
 - [ X] Include logs or screenshots of issue.

**Comment by MojoJojoDojo** [comment]
Because I dont want too many issues cluttered about this you could reply here if you got that issue in other games.

**Comment by ANR2ME** [comment]
I think you should make the list into checklist, you can copy the code from the checklist at the bottom

**Comment by MojoJojoDojo** [comment]
I said it happens without wlan being on even lol (but I tried with localhost).
I dont think its entirely  Networking related... not sure.

**Comment by ANR2ME** [comment]
what will happen on real PSP with wlan off? will an error message appeared?

**Comment by MojoJojoDojo** [comment]
With  Samurai Warriors - State Of War it will let you progress until you try to host a game.
There is no error message the game will give you an in-game message that you need to turn the WLAN on (no PSP dialog etc).
Important note here is that the menu that opens after you press versus wont turn the wlan on the PSP so yeah it might not  be networking related here at all (but it still gonna get that label lol).
EA Replay is the same it will show an internal in-game notification about turning wlan on but at least with this game it will do it right away (no another menu you can go throught here).
I wonder if this is a similar case to what happened with Resistance Retribution here I know that @shenweip fixed that last time.

**Comment by ANR2ME** [comment]
Hmm that is strange, on EA Replay if WLAN is off it will show "Not available" under "AdHoc", and it can't be selected, thus not showing any in-game message (which i think it's normal), how did you get that in-game message to shows up?
![image](https://user-images.githubusercontent.com/7974720/126088411-dfcaf877-165e-4e22-a8d5-f140f4d5f7ff.png)

**Comment by MojoJojoDojo** [comment]
You need to choose a supporter game like road rash or something (there are only 2 games i think).

**Comment by MojoJojoDojo** [comment]
Weird thing about EA Replay it uses a middleware called EA Nation that does work with other EA games...
Kinda reminds me of Driver 76 and the Quazal middleware that got you frozen on the lobby screen.

**Comment by ANR2ME** [comment]
You're right, it's probably non-adhoc, i tested Roard Rash II with WLAN disabled and have SCENET Logging to Verbose (tested with my infra test build which have verbose log), but the logs didn't show SCENET when it stuck at this screen:
![image](https://user-images.githubusercontent.com/7974720/126135860-b495f7bc-15cf-4975-bfe1-1083c6dab6ba.png)
![image](https://user-images.githubusercontent.com/7974720/126135944-94c9c9f1-3209-47d0-887b-220d00fa2272.png)

**Comment by MojoJojoDojo** [comment]
Found out this is a regression.
On 1.10.3 at least Samurai Warriors did work.
I think the issue stems from the fix to Resistance but I am not sure (someone will need to test).
@hrydgard maybe you will be interested in that (this is a non-adhoc issue).

**Comment by hrydgard** [comment]
Which fix to Resistance, can you link the issue?

**Comment by MojoJojoDojo** [comment]
For Resistance:
https://github.com/hrydgard/ppsspp/issues/13607
Also could be the Ratchet & Clank fix because it also involved a non-adhoc issue:
https://github.com/hrydgard/ppsspp/issues/13104

That's for Samurai Warriors.
Dunno about EA Replay but it can stay here because it also got a similar issue I guess.

**Comment by ANR2ME** [comment]
Might be due to unimplemented `sceKernelStopUnloadSelfModule` since @shenweip only fixed `sceKernelStopUnloadSelfModuleWithStatus` for #13607

**Comment by MojoJojoDojo** [comment]
Still don't get why it gets the networking label when its not networking related.
While this issue is getting the HLE/Kernel label https://github.com/hrydgard/ppsspp/issues/14637 even thought it looks like networking too lol.

**Comment by hrydgard** [comment]
Sorry, that was a mislabeling indeed.

**Comment by MojoJojoDojo** [comment]
> Might be due to unimplemented `sceKernelStopUnloadSelfModule` since @shenweip only fixed `sceKernelStopUnloadSelfModuleWithStatus` for #13607
> 
> Edit: Nevermind, if it was a regression probably something else

You are right @ANR2ME its seems like its a problem with my PPSSPP folder
I returned to a fresh ppsspp folder with 1.11.3 and higher build and the issue (with SW) no longer happenned.
I am confused about the setting that causes it with Saumurai Warriors though as I tried various System settings and nothing made it work...
I even deleted my ppsspp.ini file in my original folder and the savedata and that didn't help.
I know of another person that also got that issue...
I wonder from where its coming from its curious that my main PPSSPP folder is causing it...ס

**Comment by hrydgard** [comment]
Are there still enough games with this problem to have a dedicated issue? I think we can just close this and open separate issues for games that are still affected.
